dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1710,6 +1710,19 @@ def test_fiters_file_list(tmpdir, engine):
     )
     assert len(ddf2) == 0
 
+    # Make sure files list can be read with filters when they don't have the same columns order
+    pd.read_parquet(os.path.join(tmpdir, "part.4.parquet"), engine=engine)[
+        ["y", "x"]
```

Comment:
Minor comment: This makes it more clear what the intent is and also makes this test more robust to future changes in the original `df` being used 

```suggestion
        reversed(df.columns)
```

Reply:
You are right!